tests/src/test/scala/actionContainers/ActionContainer.scala

```diff
@@ -145,8 +169,7 @@ object ActionContainer {
 
     // ...find out its IP address...
     val (ip, port) =
-      if (WhiskProperties.getProperty("whisk.version.name") == "local" &&
-          WhiskProperties.onMacOSX()) {
+      if (System.getProperty("os.name").toLowerCase().contains("mac")) {
```

Comment:
@csantanapr, said to check if `DOCKER_HOST` is set here to determine if `docker-machine` is being used on a Mac.